from django.core.exceptions import ValidationError


from django_ecommerce_strategy_pattern.user_shipping_details.models import (
    UserShippingDetails,
)


def checkout(user):
    pass
